packages/remix-server-runtime/server.ts

```diff
@@ -509,6 +557,41 @@ async function handleDocumentRequest({
   }
 }
 
+async function handleResourceRequestRR(
+  serverMode: ServerMode,
+  staticHandler: StaticHandler,
+  routeId: string,
+  request: Request
+) {
+  try {
+    let response = await staticHandler.queryRoute(request, routeId);
```

Comment:
At the moment, we need to build the `dev` branch of `@remix-run/router` into the `remix/node_modules` directory for this to work, so we can pick up the changes in https://github.com/remix-run/react-router/pull/9353.

To do this (assumes your `remix` and `react-router` repos are sibling directories):

```bash
> cd remix
> yarn
> cd ../react-router/packages/router
> LOCAL_BUILD_DIRECTORY=../../../remix npx rollup -c
> cd ../../../remix
> ENABLE_REACT_ROUTER=1 yarn test:integration resource-route
```

Reply:
@jacob-ebey Curious if you have ideas on how to smooth this out.  The `dev` changes will get picked up in the next `react-router` release which I figure we'll do ~weekly or so. But in-between I'd hate to hold up this RRR work. I wonder if changesets would let us do a prerelease for a single package, or maybe we do router-only pre-releases with changes to static handler stuff

Reply:
Copied files into `./router` for the duration of the partial implementation